libs/sdk-core/src/breez_services.rs

```diff
@@ -732,18 +738,22 @@ impl BreezServices {
         });
     }
 
-    async fn start_backup_watcher(self: &Arc<BreezServices>) -> Result<()> {
+    async fn start_backup_watcher(self: &Arc<BreezServices>) -> SdkResult<()> {
         self.backup_watcher
             .start(self.shutdown_receiver.clone())
-            .await?;
+            .await
+            .map_err(|_| SdkError::ConnectFailed)?;
```

Comment:
In all startup flows I would use InitFailed.

Reply:
Done